src/components/manage/Tiles/Image/__snapshots__/View.test.jsx.snap

```diff
@@ -6,8 +6,18 @@ exports[`renders a view image component 1`] = `
 >
   <img
     alt=""
-    className="ui image"
```

Comment:
@nileshgulia1 why did you remove the semantic ui class here? That breaks the image handling in Volto.

Reply:
will undo the snapshots....

Reply:
@nileshgulia1 no! this is no solution. You broke Volto by taking away the semantic ui classes. You have to amend the "<ImageZoom" to implement the same semantic ui classes that were there before.

Reply:
@tisto <ImageZoom/> by default overrides css which we use. We have replaced our <Image/> from semantic ui which contains css classes already. The workaround could be just adding the css manually which semantic ui provides. 
https://github.com/Semantic-Org/Semantic-UI-React/blob/d6f20f90f35494a2eeeb0c1586658172fe98ea06/src/elements/Image/Image.js#L54

Reply:
You have to pass those css classes to the ImageZoom component.